Bio/GenBank/__init__.py

```diff
@@ -1369,6 +1372,9 @@ def __init__(self):
         self._cur_feature = None
         self._cur_qualifier = None
 
+    def tsa(self, content):
+        self.data.tsa = content.split('-')
+
```

Comment:
Not sure how I typed these quotes - unlike the test which was a copy-and-paste. Anyway, we've recently started enforcing double quotes:

```$ flake8
./Tests/test_GenBank.py:692:19: Q000 Remove bad quotes
./Tests/test_GenBank.py:692:72: Q000 Remove bad quotes
./Bio/GenBank/__init__.py:1376:39: Q000 Remove bad quotes
```